source/zipkin/span_context.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <vector>
+#include <string>
+
+#include "zipkin/zipkin_core_types.h"
+#include "zipkin/util.h"
+
+namespace Zipkin {
+
+typedef struct Annotation_isset_ {
+  Annotation_isset_() : cs(false), cr(false), ss(false), sr(false) {}
+  bool cs : 1;
+  bool cr : 1;
+  bool ss : 1;
+  bool sr : 1;
+} Annotation_isset_;
+
+class SpanContext {
```

Comment:
To make the code easier to read/review, please have at minimum class level comments for each class/struct. Optimally, every public function should have proper doc comments to make it easier to review. (This applies everywhere, won't comment futher).